code/modules/mob/living/update_status.dm

```diff
@@ -112,22 +112,6 @@
 /mob/living/proc/update_stamina()
 	return
 
-/mob/living/update_stat(reason = "None given")
```

Comment:
Literally every entry level base type of mob overrides this, so it's pointless for this to be here.